diff --git a/pyproject.toml b/pyproject.toml
index 61f432a..84932ae 100644
--- a/pyproject.toml
+++ b/pyproject.toml
@@ -24,8 +24,9 @@ license = "MIT"
 license-files = ["LICENSE"]
 readme = "README.rst"
 dependencies = [
-    "requests>=2.28.1",
-    "flask>=2.0.0",
+    # conda-forge: these are not used
+    # "requests>=2.28.1",
+    # "flask>=2.0.0",
     "sphinx-last-updated-by-git",
 ]
 dynamic = [
